.github/workflows/release.yml

```
@@ -159,7 +120,7 @@ jobs:
         with:
           ref: ${{ needs.create-metadata.outputs.version }}
 
-      - uses: oven-sh/setup-bun@v1
+      - uses: bun@22457c87c1b161cf7dde222c3e82b2b5f8d2bed2 # v2
```

actionlint (1.7.7)

specifying action "bun@22457c87c1b161cf7dde222c3e82b2b5f8d2bed2" in invalid format because owner is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)